recipes/recipes_emscripten/simplejson/recipe.yaml

```diff
@@ -0,0 +1,13 @@
+context:
+  version: 3.19.1
+
+package:
+  name: simplejson
+  version: "{{ version }}"
+
+build:
+  number: 0
+
```

Comment:
should we also get the `test` from the original conda-forge recipe? https://github.com/conda-forge/simplejson-feedstock/blob/main/recipe/meta.yaml#L30-L38

Reply:
you need to add such tests in the form of a `test_simplejson.py` / `test_<some_name>`.py and test the imports there.
ie:
```python
def test_imports():
    import simplejson
    import simplejson._speedups 
```